attiny84_sketch/pidog2/pidog2.ino.hex

```diff
@@ -0,0 +1,239 @@
+:100000002DC051C050C09EC5E3C54DC04CC04BC0B3
```

Comment:
I'd prefer that this file not be here at all. Let's have the hex files in the `firmware_images/` folder only.

Reply:
My mistake. I didn't realize I had added it to git.